cypress/features/regression/experimental/checkbox.feature

```diff
@@ -41,17 +41,17 @@ Feature: Experimental Checkbox component
 
```

Comment:
The two tests above should be:

`@positive
  Scenario: Enable fieldHelpInline
    When I open default "Experimental Checkbox Test" component in noIFrame with "checkbox" json from "experimental" using "fieldHelpInline" object name
    Then Checkbox field help is inline

  @positive
  Scenario: Enable and disable fieldHelpInline
    When I open default "Experimental Checkbox Test" component in noIFrame with "checkbox" json from "experimental" using "fieldHelpInlineDisabled" object name
    Then Checkbox field help is not inline`

Reply:
Changed `Experimental Checkbox` to be `Experimental Checkbox Test`. Thanks @DipperTheDan